diff --git a/CMakeLists.txt b/CMakeLists.txt
index 72fa542..855fc69 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -111,7 +111,7 @@ endif()
 if(YAML_CPP_MAIN_PROJECT)
   target_compile_options(yaml-cpp
     PRIVATE
-      $<${not-msvc}:-Wall -Wextra -Wshadow -Weffc++ -Wno-long-long>
+      $<${not-msvc}:-Wall -Wextra -Wno-shadow -Weffc++ -Wno-long-long>
       $<${not-msvc}:-pedantic -pedantic-errors>)
 endif()

